docs/sources/configuration/integrations/cloudwatch-exporter-config.md

```diff
@@ -0,0 +1,359 @@
+---
+aliases:
+
+- /docs/agent/latest/configuration/integrations/cloudwatch-exporter/ title: cloudwatch_exporter
+
+---
+
+# cloudwatch_exporter
+
+The `cloudwatch_exporter_config` block configures the `cloudwatch_exporter` integration, which is an embedded version of
+[`YACE`](https://github.com/nerdswords/yet-another-cloudwatch-exporter/). This allows the collection
+of [AWS CloudWatch](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html) metrics.
+
+This integration allows one to scrape CloudWatch metrics is a set of configurations which we will call `jobs`. There are 
+two kind of jobs: [`discovery`](#discovery_job) and [`static`](#static_job).
+
+Configuration reference:
+
+```yaml
+  autoscrape:
+    # Enables autoscrape of integrations.
+      [ enable: <boolean> | default = true ]
+
+      # Specifies the metrics instance name to send metrics to. Instance
+      # names are located at metrics.configs[].name from the top-level config.
+      # The instance must exist.
+      #
+      # As it is common to use the name "default" for your primary instance,
+      # we assume the same here.
+      [ metrics_instance: <string> | default = "default" ]
+
+      # Autoscrape interval and timeout. Defaults are inherited from the global
+      # section of the top-level metrics config.
+      [ scrape_interval: <duration> | default = <metrics.global.scrape_interval> ]
+      [ scrape_timeout: <duration> | default = <metrics.global.scrape_timeout> ]
```

Comment:
I think you pulled the v2 common config reference, I believe the OOTB params are https://github.com/grafana/agent/blob/e2d6f973e57cd5e863520560b1f5699585a444e7/pkg/integrations/config/config.go#L12-L27

I think the ES docs cover it https://github.com/grafana/agent/blob/e2d6f973e57cd5e863520560b1f5699585a444e7/docs/sources/configuration/integrations/elasticsearch-exporter-config.md#L20-L60

Reply:
indeed. Fixing!